utils/dev-tool/src/main.rs

```diff
@@ -0,0 +1,110 @@
+use std::path::PathBuf;
+use std::{env, fs};
+use structopt::StructOpt;
+
+pub mod android;
+pub mod apple;
+pub mod server;
+pub mod setup;
+pub mod utils;
+pub mod workspace;
+
+#[derive(Debug, PartialEq, StructOpt)]
+#[structopt(about = "Lockbook's development and ci tool.")]
+enum Commands {
+    // CI steps in order --------------
+    /// Check the formatting of the workspace
+    CheckWorkspaceFmt,
+
+    /// Check the lint of the workspace
+    CheckWorkspaceClippy,
+
+    /// Run the server detached
+    RunServer,
+
+    /// Run all rust tests
+    RunRustTests,
+
+    /// Check the formatting of the android client
+    CheckAndroidFmt,
+
+    /// Check the lint of the android client
+    CheckAndroidLint,
+
+    /// Run kotlin integration tests
+    RunKotlinTests,
+
+    /// Run the swift integration tests
+    RunSwiftTests,
+
+    /// Kill the server for commit hash
+    KillServer,
+
+    // End of CI steps --------------
+    /// Verify CI environment
+    VerifyCIEnvironment,
+
+    /// Make kotlin jni libs
+    MakeKotlinLibs,
+
+    /// Make kotlin jni libs for tests
+    MakeKotlinTestLib,
+
+    /// Make swift jni libs for tests
+    MakeSwiftTestLib,
+
+    /// Kill all servers running
+    KillAllServers,
+}
+
+#[derive(Clone)]
```

Comment:
`Clone` is derived for use in two places where borrowing would just work. Curious the aversion to borrowing throughout?